ts/store/reducers/wallet/wallets.ts

```diff
@@ -533,12 +545,24 @@ const reducer = (
     //
     // set favourite wallet
     //
-
+    case getType(setFavouriteWalletRequest):
+      return {
+        ...state,
+        updatingFavouriteWallet: pot.someLoading(action.payload)
+      };
```

Comment:
Can you add a comment to explain why do you use `pot.someLoading` instead of `pot.toLoading`?
In my understanding is related to the `isTheSameWallet` in the `ts/components/wallet/FavoriteMethodSwitch.tsx`, is it correct?

Reply:
Yes you are right.
But today I find a better approach following your doubts.
What do you think? This should represent better the scenario
https://github.com/pagopa/io-app/pull/3727/commits/2913953cac92fe16cdca809db0ae81a89a773dc7